src/lib/common.c

```diff
@@ -174,8 +181,9 @@ int crc32file(const char *filename, uint32_t *crc32result)
 
 	} while (!feof(file));
 
-	int rc_minor;
+	if (buf) free(buf);
```

Comment:
Maybe… just in case:

```suggestion
	if (buf) free(buf);
	buf = NULL;
```

Reply:
done